common/reviews/api/rush-lib.api.md

```diff
@@ -379,8 +397,6 @@ export class RushConfiguration {
 
 // @public
 export class RushConfigurationProject {
-    // Warning: (ae-forgotten-export) The symbol "IRushConfigurationProjectJson" needs to be exported by the entry point index.d.ts
-    //
     // @internal
     constructor(projectJson: IRushConfigurationProjectJson, rushConfiguration: RushConfiguration, tempProjectName: string);
```

Comment:
This is still internal. What would a consumer of the `rush-lib` API use `IRushConfigurationProjectJson` for? Should `IRushConfigurationProjectJson` be exported, but also marked as `@internal`?

Reply:
Really, this is for automation around Rush. I have a tool built into the monorepo, `rush scaffold` that prompts a user for inputs, builds out a new project, and adds it to `rush.json`.

As you can imagine, going from 5-50 projects has introduced a number of (scaling) challenges for my team on Rush... a lot of which is eased by automation.